qiskit/aqua/algorithms/phase_estimators/phase_estimator.py

```diff
@@ -0,0 +1,208 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""The Quantum Phase Estimation Algorithm."""
+
+
+from typing import Optional, Union
+import numpy
+from qiskit.circuit import QuantumCircuit
+import qiskit
+from qiskit.circuit.library import PhaseEstimation
+from qiskit.circuit.classicalregister import ClassicalRegister
+from qiskit.providers import BaseBackend
+from qiskit.aqua import QuantumInstance
+from qiskit.aqua.algorithms import QuantumAlgorithm
+from qiskit.aqua.utils import get_subsystem_density_matrix
+from qiskit.result import Result
+from .phase_estimator_result import PhaseEstimatorResult, _sort_phases
+
+
+class PhaseEstimator(QuantumAlgorithm):
+    """Run the Quantum Phase Estimation algorithm.
```

Comment:
The abbreviation QPE should be introduced before it is used 🙂 
```suggestion
    """Run the Quantum Phase Estimation (QPE) algorithm.
```